unit-tests/py/rspy/devices.py

```diff
@@ -93,6 +93,75 @@ def enabled( self ):
         return self._removed is False
 
 
+def disable_all( timeout = 5):
```

Comment:
Actually, now that I look at it, it's really `wait_until_no_devices`?

Reply:
`wait_until_all_ports_disabled`?